shuup/front/api/orders.py

```diff
@@ -0,0 +1,104 @@
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2017, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import mixins, serializers, viewsets
+from shuup.api.mixins import PermissionHelperMixin
+from shuup.core.api.address import AddressSerializer
+from shuup.core.api.orders import OrderFilter, OrderStatusChangeMixin
+from shuup.core.models import get_person_contact, Order, OrderLine, Shop
+
+
+class ShopSerializer(serializers.ModelSerializer):
+    logo = serializers.SerializerMethodField()
+    contact_address = AddressSerializer()
+
+    class Meta:
+        model = Shop
+        fields = ("id", "name", "logo", "options", "contact_address")
```

Comment:
`options` here should be serialized as JSON, like: https://github.com/shuup/shuup/blob/api-improvements/shuup/core/api/shop.py#L31:

```python
options = serializers.JSONField(binary=False, required=False)
```